### ISSUE TITLE
Can not create cron schedule with L character

### ISSUE THREAD

**Issue by henri5** [kagkarlsson/db-scheduler#352]
## Expected Behavior

Can create a cron schedule with pattern with L e.g. `0 * * * * 1L`

```
    @Test
    public void canUseL() {
        assertFalse(Schedules.cron("0 * * * * 1L").isDisabled());
    }
```
## Current Behavior

throws error:
```
...
Caused by: java.lang.IllegalArgumentException: Invalid chars in expression! Expression: 1L Invalid chars: L
	at com.cronutils.model.field.expression.visitor.ValidationFieldExpressionVisitor.checkUnsupportedChars(ValidationFieldExpressionVisitor.java:46)
	at com.cronutils.model.field.expression.visitor.ValidationFieldExpressionVisitor.visit(ValidationFieldExpressionVisitor.java:93)
	at com.cronutils.model.field.expression.visitor.ValidationFieldExpressionVisitor.visit(ValidationFieldExpressionVisitor.java:25)
	at com.cronutils.model.field.expression.On.accept(On.java:70)
	at com.cronutils.model.SingleCron.validate(SingleCron.java:79)
	at com.cronutils.parser.CronParser.parse(CronParser.java:163)
	... 72 more
```

Seems that this broke with https://github.com/jmrozanec/cron-utils/issues/370, using CronType.SPRING53 would work. Pattern worked with older version of db-scheduler (tested 6.2) possibly broke with https://github.com/kagkarlsson/db-scheduler/commit/596095dc76bc0e687dc51f317e7fbfb17ae232d8

**Comment by kagkarlsson** [comment]
Ok, interesting. Maybe we need to make it configurable.. 🤔

**Comment by kagkarlsson** [comment]
Should be fixed in `v12.0.1`
